addons/website_event_track/models/event_track_stage.py

```diff
@@ -18,12 +18,19 @@ class TrackStage(models.Model):
     fold = fields.Boolean(
         string='Folded in Kanban',
         help='This stage is folded in the kanban view when there are no records in that stage to display.')
-    is_accepted = fields.Boolean(
-        string='Accepted Stage',
-        help='Accepted tracks are displayed in agenda views but not accessible.')
-    is_done = fields.Boolean(
-        string='Done Stage',
-        help='Done tracks are automatically published so that they are available in frontend.')
-    is_cancel = fields.Boolean(string='Canceled Stage')
-    is_done = fields.Boolean()
+    status = fields.Selection([
```

Comment:
This seems way too much complicated to me. We will need to rediscuss this point with LNA as I fear we went too far in details. Globally I understand the requirement; however Odoo should stay as simple as possible for end users and this is going too much into details. To re-spec I think.